server/src/test/java/org/apache/iotdb/db/engine/memtable/MemTableFlushTaskTest.java

```diff
@@ -37,7 +37,7 @@
 
   private RestorableTsFileIOWriter writer;
   private String storageGroup = "storage_group1";
-  private String filePath = "data/testUnsealedTsFileProcessor.tsfile";
+  private String filePath = "target/tmpdir/data/testUnsealedTsFileProcessor.tsfile";
```

Comment:
```suggestion
  private String filePath = TestConstant.OUTPUT_DATA_DIR + File.separator + "testUnsealedTsFileProcessor.tsfile";
```
add a output-data-dir as   target + File.separator + data

This test is failed in my computer because the target/tmpdir/data is not created. just put the file under target/data is ok